backend/src/v1/complaint/complaint.service.ts

```diff
@@ -1040,7 +1046,10 @@ export class ComplaintService {
       }
 
       // -- filter by complaint identifiers returned by case management if outcome animal filter is present
-      if (agency === "COS" && (filters.outcomeAnimal || filters.outcomeAnimalStartDate)) {
+      if (
+        (agencies.includes("EPO") || agencies.includes(Role.PARKS)) &&
```

Comment:
I think this needs to be COS or PARKS, not EPO. Filtering by outcome by animal isn't doing anything for me on this branch (the filter gets added but it doesn't filter the results, tested as both COS & PARKS), other filters seem to work as intended